doc/build.md

```diff
@@ -1,35 +1,31 @@
-# Build instructions for Linux
+# Build instructions
```

Comment:
These instructions may work for Mac, but need to be validated. Bazel gained native support for Mac recently.

Reply:
This is stated in the Environment section. Putting "for Linux" in the heading seems like repetition for me.